metatomic-torch/src/register.cpp

```diff
@@ -249,12 +254,30 @@ TORCH_LIBRARY(metatomic, m) {
         /*name=*/"register_autograd_neighbors",
         /*overload_name=*/"register_autograd_neighbors",
         /*arguments=*/{
-            c10::Argument("system", c10::getTypePtr<metatomic_torch::System>()),
+            c10::Argument("system", c10::getTypePtr<System>()),
             c10::Argument("neighbors", c10::getTypePtr<metatensor_torch::TensorBlock>()),
             c10::Argument("check_consistency", c10::getTypePtr<bool>(), c10::nullopt, /*default_value=*/false),
         },
         /*returns=*/{}
     );
     schema.setAliasAnalysis(c10::AliasAnalysisKind::CONSERVATIVE);
     m.def(std::move(schema), register_autograd_neighbors);
+
+    m.def("save_system_buffer(__torch__.torch.classes.metatomic.System system) -> Tensor", 
+        [&](const System& system) { return save_buffer(system); }
+    );
+    m.def("load_system_buffer(Tensor buffer) -> __torch__.torch.classes.metatomic.System", 
+        [&](const torch::Tensor& buffer) -> System { return load_system_buffer(buffer); }
+    );
+
+    m.def("save_system_file(str path, __torch__.torch.classes.metatomic.System system) -> ()", 
+        [&](const std::string& path, const System& system) {
+            save(path, system);
+        }
+    );
+    m.def("load_system_file(str path) -> __torch__.torch.classes.metatomic.System", 
+        [&](const std::string& path) -> System {
+            return load_system(path);
+        }
+    );
```

Comment:
I think these should have the same name as everywhere


```suggestion
    m.def("save_buffer(__torch__.torch.classes.metatomic.System system) -> Tensor", 
        [&](const System& system) { return save_buffer(system); }
    );
    m.def("load_system_buffer(Tensor buffer) -> __torch__.torch.classes.metatomic.System", 
        [&](const torch::Tensor& buffer) -> System { return load_system_buffer(buffer); }
    );

    m.def("save(str path, __torch__.torch.classes.metatomic.System system) -> ()", 
        [&](const std::string& path, const System& system) {
            save(path, system);
        }
    );
    m.def("load_system(str path) -> __torch__.torch.classes.metatomic.System", 
        [&](const std::string& path) -> System {
            return load_system(path);
        }
    );
```